tests/test_resource.py

```diff
@@ -0,0 +1,12 @@
+from tutor.models.course import Resource
+
+
+def test_add_new_resource():
+    client = app.test_client()
+    assert Resource.query.all() == []
+    response = client.post('/resource', data=dict(
```

Comment:
There is no '/resource` route. If you want to add a new resource you should call `/resource/new?id=%COURSE_ID%. As on `/tutor/routes/resoruces.py`. 
By the way, that kind of test already implemented on #62 . Maybe you can focus on testing the model itself, as I did on `Model` test, instead the route?